python-checks/src/main/java/org/sonar/python/checks/FloatingPointEqualityCheck.java

```diff
@@ -38,20 +43,29 @@
 public class FloatingPointEqualityCheck extends PythonSubscriptionCheck {
 
   private static final String MESSAGE = "Do not perform equality checks with floating point values.";
-  private static final String QUICK_FIX_MESSAGE = "Replace with %smath.isclose().";
+  private static final String QUICK_FIX_MESSAGE = "Replace with %s%s.isclose().";
 
-  private static final String QUICK_FIX_REPLACE = "%smath.isclose(%s, %s, rel_tol=1e-09, abs_tol=1e-09)";
+  private static final String QUICK_FIX_MATH = "%s%s.isclose(%s, %s, rel_tol=1e-09, abs_tol=1e-09)";
+
+  private static final String QUICK_FIX_IMPORTED_MODULE = "%s%s.isclose(%s, %s, rtol=1e-09, atol=1e-09)";
 
   private static final Tree.Kind[] BINARY_OPERATION_KINDS = { Tree.Kind.PLUS, Tree.Kind.MINUS, Tree.Kind.MULTIPLICATION,
       Tree.Kind.DIVISION };
 
   private ReachingDefinitionsAnalysis reachingDefinitionsAnalysis;
+  private static final List<String> SUPPORTED_IS_CLOSE_MODULES = Arrays.asList("math", "numpy", "torch");
+
+  private String importedModuleForIsClose;
+  private Name importedAlias;
```

Comment:
Note that the same Check object is used throughout the analysis: if those names are set during the analysis of one file, they will not be reinitialized in the next file, and we might erroneously assume names are imported when they're not.